conf/cortx_rgw.conf

```diff
@@ -3,13 +3,21 @@
 
 [client]
     rgw backend store = motr
+
+[client.radosgw-admin]
+    motr profile fid =
+    motr ha endpoint =
+    motr my endpoint =
+    motr my fid =
+    motr admin fid =
+    motr admin endpoint =
+    log file =
+
```

Comment:
Remove extra line or be consistent in spaces between the stanzas.

Reply:
done.